program/src/instruction.rs

```diff
@@ -69,18 +69,20 @@ pub enum TokenWrapInstruction {
     ///
     /// Accounts expected by this instruction:
     ///
-    /// 0. `[writeable]` Wrapped token account to unwrap
-    /// 1. `[writeable]` Wrapped mint, address must be:
+    /// 0. `[writeable]` Escrow of unwrapped tokens, must be owned by:
+    ///    `get_wrapped_mint_authority(wrapped_mint_address)`
+    /// 1. `[writeable]` Recipient unwrapped tokens
+    /// 2. `[]` Wrapped mint authority, address must be:
+    ///    `get_wrapped_mint_authority(wrapped_mint)`
+    /// 3. `[]` Unwrapped token mint
+    /// 4. `[]` SPL Token program for wrapped mint
+    /// 5. `[]` SPL Token program for unwrapped mint
+    /// 6. `[writeable]` Wrapped token account to unwrap
+    /// 7. `[writeable]` Wrapped mint, address must be:
     ///    `get_wrapped_mint_address(unwrapped_mint_address,
     ///    wrapped_token_program_id)`
-    /// 2. `[writeable]` Escrow of unwrapped tokens, must be owned by:
-    ///    `get_wrapped_mint_authority(wrapped_mint_address)`
-    /// 3. `[writeable]` Recipient unwrapped tokens
-    /// 4. `[]` Unwrapped token mint
-    /// 5. `[]` SPL Token program for wrapped mint
-    /// 6. `[]` SPL Token program for unwrapped mint
-    /// 7. `[signer]` Transfer authority on wrapped token account
-    /// 8. `..8+M` `[signer]` (Optional) M multisig signers on wrapped token
+    /// 8. `[signer]` Transfer authority on wrapped token account
+    /// 9. `..8+M` `[signer]` (Optional) M multisig signers on wrapped token
```

Comment:
Re-arranged in order so that accounts can be sub-sliced intothe transfer_checked account slice